public/javascripts/loginScripts.js

```diff
@@ -1,43 +1,52 @@
-$('.form').find('input, textarea').on('keyup blur focus', function (e) {
-
-      var $this = $(this),
-          label = $this.prev('label');
-
-          if (e.type === 'keyup') {
-                if ($this.val() === '') {
-              label.removeClass('active highlight');
-            } else {
-              label.addClass('active highlight');
-            }
-        } else if (e.type === 'blur') {
-            if( $this.val() === '' ) {
-                label.removeClass('active highlight');
-                } else {
-                label.removeClass('highlight');
-                }
-        } else if (e.type === 'focus') {
-
-          if( $this.val() === '' ) {
-                label.removeClass('highlight');
-                }
-          else if( $this.val() !== '' ) {
-                label.addClass('highlight');
-                }
-        }
-
-    });
-
-    $('.tab a').on('click', function (e) {
-
-      e.preventDefault();
-
-      $(this).parent().addClass('active');
-      $(this).parent().siblings().removeClass('active');
-
-      target = $(this).attr('href');
-
-      $('.tab-content > div').not(target).hide();
-
-      $(target).fadeIn(600);
-
-    });
+$('.loginForm').find('input, textarea').on('keyup blur focus', function (event) {
+
+  var $this = $(this),
+      label = $this.prev('label');
+
+  if (event.type === 'keyup') {
+        if ($this.val() === '') {
+      label.removeClass('active highlight');
+  } else {
+    label.addClass('active highlight');
+  }
+} else if (event.type === 'blur') {
+    if( $this.val() === '' ) {
+      label.removeClass('active highlight');
+    } else {
+      label.removeClass('highlight');
+    }
+  } else if (event.type === 'focus') {
+    if( $this.val() === '' ) {
+      label.removeClass('highlight');
+    }
+    else if( $this.val() !== '' ) {
+      label.addClass('highlight');
+    }
+  }
+});
+
+$('.tab a').on('click', function (event) {
+  event.preventDefault();
+  $(this).parent().addClass('active');
+  $(this).parent().siblings().removeClass('active');
+  target = $(this).attr('href');
```

Comment:
avoid using global variables. I don't see where it's declared, but if it is, ignore this